### DISCUSSION TITLE
FEAT: Canvas' "Back to top"

### DISCUSSION THREAD

**Discussion by mcottret** [GrapesJS/grapesjs#3545] [Ideas]
**What are you trying to add to GrapesJS?**

So our team is planning to add a "Back to top" feature in our editor implementation, allowing users to quickly go back to the top of the Canvas when working on big pages. We were wondering if it could be of interest to develop it as an addition to GJS rather than on our own.

This could include: 

- A "Back to top" command
- An init configuration allowing to enable / disable the feature, as well as configure it's behaviour (from the top of my head options could include configuring a "Back to top" button's position / style / element / show | hide behaviour (eg by scroll %, or custom) ...)
- ... anything else that could be relevant (eg allowing to remember or reset scroll position between pages)

@artf do you think it would be valuable to integrate this feature into GJS' core (or a plugin) ? If yes, do you have any specific idea or guidance ?

Thanks for your attention :)

**Is there an alternative at the latest version?**

 - [ ] Yes
 - [x] No 

**Is this related to an issue?**

 - [ ] Yes
 - [x] No

**Comment by artf** [comment]
As a core, I don't think so, but for sure it could be a nice plugin to have :)
